service/src/main/java/com/ge/predix/acs/privilege/management/dao/ResourceRepositoryProxy.java

```diff
@@ -26,10 +26,18 @@
     @Autowired(required = false)
     private GraphResourceRepository graphRepository;
 
+    public GraphResourceRepository getGraphRepository() {
```

Comment:
Please make these methods package private.  Should not be needed by services or any other using "layers"